cgroups/cgroups_v1.go

```diff
@@ -526,6 +588,14 @@ func (cg *CgroupV1) GetNetworkData(pid int, cgPath string) (*rspec.LinuxNetwork,
 
 // GetPidsData gets cgroup pids data
 func (cg *CgroupV1) GetPidsData(pid int, cgPath string) (*rspec.LinuxPids, error) {
+	if filepath.IsAbs(cgPath) {
+		path := filepath.Join(cg.MountPath, "pids", cgPath)
+		if _, err := os.Stat(path); err != nil {
+			if os.IsNotExist(err) {
+				return nil, specerror.NewError(specerror.CgroupsAbsPathRelToMount, fmt.Errorf("In the case of an absolute path, the runtime MUST take the path to be relative to the cgroups mount point"), rspec.Version)
+			}
```

Comment:
the last place.
Besides these, it looks good to me.

Reply:
updated, PTAL.